docs/user-guide/story-setting.md

```diff
@@ -1,13 +1,17 @@
+
  
- 
-The story settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
+The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
```

Comment:
The Story Settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in Edit Mode and it can be opened by clicking on the **Settings** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.